pmm-app/cypress/integration/e2e/qan/filters_spec.js

```diff
@@ -0,0 +1,38 @@
+import { baseUrl, password, qanUrl, username } from '../../../../config'
```

Comment:
I'd encourage to avoid using underscores in file names. The current best practice suggests to use dashes or dots instead.
If this is a `spec` file, I'd suggest to name it `/filters.spec.js`.